streams/runners/storm/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/storm/testing/TestRunSinkBolt.java

```diff
@@ -72,20 +71,13 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
             LOG.error("Can't open file for preparing to write: " + outputFilePath);
             throw new RuntimeException(e);
         }
-
-        eventLogger = TestRunEventLogger.getEventLogger(testRunSink.getEventLogFilePath());
     }
 
     @Override
     public void execute(Tuple input) {
         Object value = input.getValueByField(StreamlineEvent.STREAMLINE_EVENT);
-        String sourceComponentId = input.getSourceComponent();
-        String streamId = input.getSourceStreamId();
         StreamlineEvent event = (StreamlineEvent) value;
 
-        eventLogger.writeEvent(System.currentTimeMillis(), TestRunEventLogger.EventType.INPUT, testRunSink.getName(),
-                streamId, StormTopologyUtil.extractStreamlineComponentName(sourceComponentId), event);
-
         String outputFilePath = testRunSink.getOutputFilePath();
         try (FileWriter fw = new FileWriter(outputFilePath, true)) {
```

Comment:
Seems we are writing the events to this file in the sink. Why is it required?

Reply:
This is for aggregating the result of sinks and storing the aggregated result to test history entity. We may still want to sort out features for test run and remove unnecessary features.